doc/user_guide/dynamo0p3.rst

```diff
@@ -150,20 +150,20 @@ the field is on. Placement of field data points, also called degrees of
 freedom (hereafter "DoFs"), is determined by the function space the field
 is on.
 LFRic fields can have ``real``-valued data or ``integer``-valued data.
-Properties of the ``real``-valued fields in the LFRic infrastructure
-are stored in the ``field_type`` class, and of the ``integer``-valued
-fields in the ``integer_field_type`` class.
+In the LFRic infrastructure, these fields are represented by instances of
+the ``field_type`` and ``integer_field_type`` classes, respectively.
 
 .. _dynamo0.3-field-vector:
 
 Field Vector
 ++++++++++++
 
-Depending on the function space a field lives on, the field data at a
-point can be a scalar or a vector (see :ref:`dynamo0.3-function-space`
+Depending on the function space a field lives on, the field data value at
+a point can be a scalar or a vector (see :ref:`dynamo0.3-function-space`
 for the list of scalar and vector function spaces). There is an
-additional option which specifies whether the data itself is vector
-valued and it usually refers to bundles of scalar-valued fields.
+additional option, called a *field vector*, to represent a bundle of
+either scalar- or vector-valued fields (in LFRic it is commonly used
```

Comment:
Thanks for tightening this up. It's clear to me what it means now :-) However, since this is PSyclone documentation, we don't need to talk about LFRic usage so I think we can drop the extra text completely (i.e. the bit in parentheses).

Reply:
Dropped the bit in parentheses.